src/test/java/uk/gov/justice/digital/delius/service/OffenderManagerService_allocatePrisonOffenderManagerTest.java

```diff
@@ -80,502 +82,588 @@ public void setup() {
                 referenceDataService,
```

Comment:
I have only added one test here, the huge change is simple due to moving all tests into a `@Nested` class for two main service methods.  

Look for new test down the page.